### DISCUSSION TITLE
Feature request (vue-query):Support for devtool embedded in nuxt3's devtool

### DISCUSSION THREAD

**Discussion by MangMax** [TanStack/query#8110] [Ideas]
Hello,
 I'm currently working with Nuxt 3 and I am looking for a way to integrate TanStack Query Devtools directly into the Nuxt 3 development panel. The goal is to streamline my workflow by avoiding the need to switch between multiple panels.
I understand that there's an option to add third-party panels to Nuxt, and I've attempted this with TanStack Query Devtools but haven't succeeded yet. I was wondering if anyone in the community has managed to accomplish this integration and would be willing to share their insights or provide guidance.
Your assistance would be greatly appreciated, and I thank you in advance for any help you can provide.

**Comment by sevensolutions** [comment]
This would be really cool. 🙏
